templates/btax/includes/results/table.html

```diff
@@ -0,0 +1,57 @@
+{% load humanize %}
```

Comment:
in PR #309 `templates/btax/includes/results` has been removed (in reference to this comment and comment above)